h/db/__init__.py

```diff
@@ -153,6 +135,27 @@ def _maybe_create_default_organization(engine, authority):
     session.commit()
     session.close()
 
+    return default_org
+
+
+def _maybe_create_world_group(engine, authority, default_org):
+    from h import models
+    from h.models.group import ReadableBy, WriteableBy
+    session = Session(bind=engine)
+    world_group = session.query(models.Group).filter_by(pubid='__world__').one_or_none()
+    if world_group is None:
+        world_group = models.Group(name=u'Public',
+                                   authority=authority,
+                                   joinable_by=None,
+                                   readable_by=ReadableBy.world,
+                                   writeable_by=WriteableBy.authority,
+                                   organization=default_org)
```

Comment:
Adding the `organizatiopn` argument here and the `default_org` parameter above are the only two changes to `_maybe_create_world_group()` but I also moved it to below `_maybe_create_default_organization()` in the file since it's now called after `_maybe_create_default_organization()`.